src/core/utils/Predicates.ts

```diff
@@ -0,0 +1,15 @@
+import { VNode } from '../stores/VNode';
+import { VNodeType } from '../stores/VNodeType';
+
+const RangeTypes = [VNodeType.RANGE_TAIL, VNodeType.RANGE_HEAD];
+export const isRange = (node: VNode): boolean => {
```

Comment:
Docstrings ?

Reply:
I would write it with the `function` keyword otherwise the function is anonymous and it makes it more complicated to debug. 
```typescript
export function isRange(node: VNode): boolean => {
//...
}
```